doc/1/controllers/document/m-delete/index.md

```diff
@@ -0,0 +1,44 @@
+---
+code: true
+type: page
+title: MDeleteAsync
+description: Deletes multiple documents.
+---
+
+# MDeleteAsync
+
+Deletes multiple documents.
+
+Throws a partial error (error code 206) if one or more document deletions fail.
+
+## Arguments
+
+```csharp
+public async Task<JArray> MDeleteAsync(
+  string index, 
+  string collection, 
+  JArray ids, 
```

Comment:
Aaaaaand now that I see the documentation, I think that using a `JArray` here is a mistake, and that we should ask for a `string[]` instead.

Sorry, my fault. :expressionless:

Reply:
What do we do ? Fix in this PR and add breaking-change tag?

Reply:
I'm wondering that myself. You know what: let it like that, I'll create a branch starting from that one and fix the problem. After all, I'm the cause of it. :expressionless: